Linked List Implementation of Stack

Arun9739 opened the pull request:
## Related Issue
- Stack Using Linked List (Issue No. #352 )

## Proposed Changes
- I have uploaded the file which implements stack using linked list

## Additional Info
-  In this program I have used a head as meta data structure for the Stack. 
-  We can see the functions for creating stack , pushing an item into the stack, poping the item from the stack, getting the stack count, getting the stack top, checking the stack is empty etc.
-  The nodes used in the functions have been allocated using the dynamic memory allocation.
-  The program is designed in such a way that we can use it for any data type such as int, float, short, double etc. Because the program has used the void pointer in its all functions.
- Just to demonstrate the concept we have used int data type in main function.

## Checklist

<!--- Go over all the following points, and put an `x` in all the boxes that apply. -->
<!--- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->
- [x] ✅ My code follows the code style of this project.
- [x] 📝 My change requires a change to the README.
- [x] 🎀 I have updated the README accordingly.
- [x] 👀 I have read the [**CONTRIBUTING**](https://github.com/Lakhankumawat/LearnCPP/blob/main/Contributing.md) document.
- [x] ✨ I have added tests to cover my changes.
- [x] 🚩 All new and existing tests passed.
- [x] 🌟 ed the repo

<!--- Describe your changes in detail -->

<!--- Provide a general summary of your changes in the Title above -->

<!--- Why is this change required? What problem does it solve? -->

<!--- If it fixes an open issue, please link to the issue here. -->

<!--- Please describe in detail how you tested your changes. -->

<!--- Include details of your testing environment, and the tests you ran to -->

<!--- see how your change affects other areas of the code, etc. -->

<!--- What types of changes does your code introduce? Put an `x` in all the boxes that apply: -->
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Output Screenshots
| Screenshot #1      | 
| ----------- |

| ![image](https://user-images.githubusercontent.com/84305637/157659020-09e24a64-d3bc-4300-8412-1a153edda404.png)  | Image goes here     |

miraj0507 commented:
Hello, @Arun9739 i dont see the readme file with your code, kindly update the readme, and tag me will look at your code after readme.

Arun9739 commented:
@miraj0507 I have updated the Readme accordingly as per your guidelines. I haven't created new Readme.md.  I have just updated the existing stack folder Readme. where my cpp file goes.
Pleasae have a look.

Arun9739 commented:
@siddhi-244 @miraj0507 Please look at my pr

Lakhankumawat commented on S-Stack/README.md:
```diff
@@ -13,4 +13,43 @@ Applications of stack :
+
+
+
+# LINKED LIST IMPLEMENTATION OF A STACK
```
Change headline to `Linked List Implamentation of Stack`

Arun9739 replied:
changed sir

Arun9739 commented:
sir please merge my pr

Arun9739 commented:
@miraj0507 @Lakhankumawat please merge my pr
